src/Components/Blazor/BlazorExtension/src/source.extension.vsixmanifest

```diff
@@ -2,8 +2,8 @@
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
         <Identity Id="Microsoft.VisualStudio.BlazorExtension" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
-        <DisplayName>ASP.NET Core Blazor Language Services</DisplayName>
-        <Description xml:space="preserve">Provides Visual Studio support for ASP.NET Core Blazor</Description>
+        <DisplayName>Blazor</DisplayName>
+        <Description xml:space="preserve">Provides Visual Studio support for Blazor</Description>
```

Comment:
Suggestion `Blazor Templates` and `Provides templates for Blazor` or similar?

Reply:
I prefer to keep it generic. This is the VSIX you need for all things Blazor. Also, I think the shorter display name "Blazor" looks more official.

Reply:
I think it makes it look less official 😁  but still better.